javascript/javascript2/week2/exercise.js

```diff
@@ -0,0 +1,76 @@
+/**
```

Comment:
Where is this file from? I can't find it in [the homework](https://github.com/HackYourFuture-CPH/JavaScript/blob/master/javascript2/week2/homework.md)